### DISCUSSION TITLE
no space left on device

### DISCUSSION THREAD

**Discussion by b11r1m** [projectdiscovery/nuclei#5385] [Q&A]
Hi team i guess this is a new bug in nuclei :
ERROR::

[WRN] [CVE-2011-5181] Could not execute step: [:RUNTIME] got following errors while executing flow <- [:RUNTIME] failed to execute http:1 protocol <- [:RUNTIME] got err while executing http://127.0.0.1/wp-content/plugins/clickdesk-live-support-chat/readme.txt <- GET http://127.0.0.1/wp-content/plugins/clickdesk-live-support-chat/readme.txt giving up after 2 attempts: Get "http:/127.0.0.1/wp-content/plugins/clickdesk-live-support-chat/readme.txt": write /tmp/nuclei1026774131/000001.log: no space left on device

**Comment by tarunKoyalwar** [comment]
seems to be a linux / environment issue and not related to nuclei ( see: https://community.grafana.com/t/how-to-fix-the-error-there-is-no-space-left-on-the-device-but-there-are-still-plenty-of-disks-available/102314/15 )

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
